client/packages/purchasing/src/purchase_order/ListView/ListView.tsx

```diff
@@ -66,107 +62,120 @@ const ListView = () => {
     setDisabledRows(disabledRows);
   }, [data, setDisabledRows]);
 
-  const columns = useColumns<PurchaseOrderRowFragment>(
-    [
-      GenericColumnKey.Selection,
+  const mrtColumns = useMemo(
+    (): ColumnDef<PurchaseOrderRowFragment>[] => [
+      {
+        header: t('label.supplier'),
+        accessorKey: 'supplier',
+        accessorFn: row => row.supplier?.name,
+        enableColumnFilter: true,
+        defaultHideOnMobile: true,
+        enableSorting: true,
+      },
+      {
+        header: t('label.number'),
+        accessorKey: 'number',
+        columnType: ColumnType.Number,
+        size: 110,
+        enableSorting: true,
+      },
+      {
+        header: t('label.created'),
+        accessorKey: 'createdDatetime',
+        columnType: ColumnType.Date,
+        enableSorting: true,
```

Comment:
I'd set a `size` on this and the other date columns so it's not unnecessarily wide